R/add_sda_target.R

```diff
@@ -1,6 +1,6 @@
 #' Add targets of CO2 emissions at the portolio level, using the SDA approach
 #'
-#' This function calculates emissions-factor targets at the portfolio-level. It
+#' This function calculates targets of CO2 emissions at the portfolio-level. It
```

Comment:
I removed the word 'factor' from 'emissions-factor' because it means nothing to me, and instead I added the word 'CO2". This may be inaccurate but is clearer to me. Undo this if most readers should immediately know what 'emission-factors' mean; else I suggest you keep it like this or include an explanation (or a link to it).

Reply:
It doesn’t calculate emissions, the targets are for the “emissions factors” which is short for “emissions intensity factor” or “CO2 emissions per unit production”. Not sure if you prefer any of those synonyms, but it’s important to specify that we aren’t speaking about absolute emissions

Reply:
What I find more meaningful is "CO2 emissions per unit production". I understand every word in isolation and they make sense together as a phrase.

I find the word "factor" too vague. It could mean a million things unless it is defined immediately before the first time you use it.

In the article you wrote:
* I fail to match any reference to "emissions factor".
* I do match "emission intensity"  in "The company-level emission intensity target is then defined as ..."
* I also match "unit of production" in "Hence to reflect this difference, a rate of change, $g(t)$ is taken from the average sector CO2-intensity per unit of production in a specific year ..."